src/providers/extraction_providers/visionExtractionProvider.ts

```diff
@@ -5,34 +5,32 @@ import { AssignedLabel, DetectedObject, Coordinate } from '../../models/data_mod
 import * as vision from '@google-cloud/vision';
 
 const enum requestTypes {
-	labelDetection = 'LABEL_DETECTION',
-	objectLocalization = 'OBJECT_LOCALIZATION',
+	LabelDetection = 'LABEL_DETECTION',
+	ObjectLocalization = 'OBJECT_LOCALIZATION',
 }
 
+type LocalizedObjectAnnotation = vision.protos.google.cloud.vision.v1.ILocalizedObjectAnnotation;
+type EntityAnnotation = vision.protos.google.cloud.vision.v1.IEntityAnnotation;
+type NormalizedVertex = vision.protos.google.cloud.vision.v1.INormalizedVertex;
+
 export class VisionExtractionProvider implements ExtractionProvider {
-	protected createAssignedLabel(
-		label: vision.protos.google.cloud.vision.v1.IEntityAnnotation
-	): AssignedLabel {
+	protected createAssignedLabel(label: EntityAnnotation): AssignedLabel {
 		return {
 			description: label?.description || '',
 			confidence: label?.score || 0,
 		};
 	}
 
-	protected createBoundingPoly(
-		boundingPoly: vision.protos.google.cloud.vision.v1.INormalizedVertex
-	): Coordinate {
+	protected static createBoundingVertex(coordinate: NormalizedVertex): Coordinate {
 		return {
-			x: boundingPoly?.x || 0,
-			y: boundingPoly?.y || 0,
+			x: coordinate?.x || 0,
```

Comment:
Wait, so the built in Vertex type from Google is nullable? Why would Google do this to us...

The default of null => (0, 0) doesn't really make sense in our use case; it probably makes more sense to filter the points out of the list altogether.